sdk/servicebus/azure-servicebus/stress/Dockerfile

```diff
@@ -0,0 +1,10 @@
+# syntax=docker/dockerfile:1
+
+FROM python:3.8-slim-buster
+
+WORKDIR /app
+
+COPY ./scripts /app/stress/scripts
+
+WORKDIR /app/stress/scripts
+RUN pip3 install -r dev_requirements.txt
```

Comment:
It must be nice to not have a build step :)

(I have to do a two-stage dockerfile for Go)